docs/module.rst

```diff
@@ -36,8 +42,8 @@ Package configuration file
 --------------------------
 
 A package configuration file can be placed at the root of the source
-tree of a library (or package) under development. It should be named
-``lambdapi.pkg``, and contain the following fields (an example is given
+tree of a library (or package) under development. Its name *must* be
+``lambdapi.pkg``, and it must contain the following fields (an example is given
```

Comment:
why not directly writing: A package configuration file ``lambdapi.pkg`` can be placed at the root of the source tree of a library (or package) under development. It must contain the following fields (an example is given...